dbt_airflow_factory/tasks_builder/builder.py

```diff
@@ -18,6 +19,35 @@
 from dbt_airflow_factory.tasks_builder.graph import DbtAirflowGraph
 
 
+@dataclass
+class ResultTaskData:
+    nodes_data: ModelExecutionTask
+    schema: str
+    model_name: str
+
+
+class TaskConnectionMapper:
+    @staticmethod
+    def map_connections(
+        result_tasks: Dict[str, ResultTaskData], dbt_airflow_graph: DbtAirflowGraph
+    ) -> NoReturn:
+        raise NotImplementedError(
+            "Connection mapper object has to implement map_connections method"
+        )
+
+
+class DefaultConnectionMapper:
```

Comment:
Shouldn't be class DefaultConnectionMapper(TaskConnectionMapper): ?

Why there is only one implementation? I thought you need an extra one?